instrumentation/jvm-metrics/src/main/java/com/splunk/opentelemetry/instrumentation/jvmmetrics/otel/OtelJvmGcMetrics.java

```diff
@@ -107,29 +107,29 @@ public void install() {
     maxDataSize = new AtomicLong((long) maxLongLivedPoolBytes);
     meter
         .gaugeBuilder("runtime.jvm.gc.max.data.size")
-        .setUnit("bytes")
+        .setUnit("By")
```

Comment:
This is one of the classes copied from micrometer, right? Shouldn't we avoid changing metrics in these?

Reply:
Good point, I'll close this pr

Reply:
I think it's fine to change the unit for the other two, "our" classes